packages/web/components/Modal/BuyTechnologyModal/index.js

```diff
@@ -0,0 +1,207 @@
+import React, { useReducer } from 'react';
+import PropTypes from 'prop-types';
+import { AiOutlineArrowLeft, AiOutlineArrowRight } from 'react-icons/ai';
+import Select from 'react-select';
+import { ReviewButton as Button } from '../CurateTechnologyModal/styles';
+import { useModal } from '../../../hooks';
+import { RequiredIndicator } from '../../Form';
+import * as S from './styles';
+
+const technologyUses = [
+	{
+		id: 1,
+		label: 'Privado',
+		value: 'private',
+	},
+	{
+		id: 2,
+		label: 'Empresa',
+		value: 'company',
+	},
```

Comment:
Not sure if this something related to the code review itself, but aren't those options the same? I mean...private and company are both private... 🤔 

if it makes sense, we can remove one of them. Any thoughts, @JoaoPauloSMoura?

Reply:
> Not sure if this something related to the code review itself, but aren't those options the same? I mean...private and company are both private... 🤔
> 
> if it makes sense, we can remove one of them. Any thoughts, @JoaoPauloSMoura?

Private is related to personal use, and company is for commercial use.

Reply:
Ty @JoaoPauloSMoura

Reply:
Got it, @JoaoPauloSMoura . Makes sense 👍